src/main/java/com/gdschongik/gdsc/domain/event/domain/EventParticipationDomainService.java

```diff
@@ -11,6 +11,9 @@
 @DomainService
 public class EventParticipationDomainService {
 
+    /**
+     * 회원이 온라인을 통해 이벤트에 참여 신청하는 메서드입니다.
```

Comment:
정회원이라는 표현을 많이 사용해왔어서 아래처럼 수정하면 좋을 것 같습니다

```suggestion
     * 정회원이 온라인으로 이벤트에 참여 신청하는 메서드입니다.
```